volcli/commands.go

```diff
@@ -34,6 +34,7 @@ var Commands = []cli.Command{
 		Subcommands: []cli.Command{
 			{
 				Name:        "upload",
+				Flags:       VolmasterFlags,
```

Comment:
actually +1 to this too. It can be provided for every command so it's no longer necessary.